.dotnet.azure/src/Custom/Assistants/AzureMessagesPageEnumerator.cs

```diff
@@ -0,0 +1,51 @@
+using System.ClientModel;
+using System.ClientModel.Primitives;
+
+#nullable enable
+
+namespace Azure.AI.OpenAI.Assistants;
+
+internal partial class AzureMessagesPageEnumerator : MessagesPageEnumerator
```

Comment:
nit: I recently moved the enumerator types in the OpenAI library to separate "Internal/Pagination" folders. Consider adopting that pattern here as well if you think it's useful. 😊

Reply:
Moving for consistency!